bin/ts-query

```diff
@@ -22,6 +22,4 @@ source bin/require_sed
 #         body: (expression_statement
 #           (integer))))
 
-bin/generate-parser 1>/dev/null
-
-npx tree-sitter parse $@ | $sed -e 's/ \[.*\]//'
+bin/generate-parser 1>/dev/null && { npx tree-sitter parse $@ | $sed -e 's/ \[.*\]//'; }
```

Comment:
What's the difference here? I think `set -e` already exists early if `bin/generate-parser` fails. Isn't `&&` achieving the same thing? Is there a different in exit codes? Or something else?

Reply:
Never mind. You answered above!

Reply:
Hmm if I add an `exit 1` to `bin/generate-parser` and run `bin/ts-query` without changes, it seems like the error code propagates. Maybe I'm missing something (also my understanding of exit codes is limited 😁).

https://user-images.githubusercontent.com/1281153/124953665-3295d480-dfca-11eb-911c-3919cf5ff656.mov

Reply:
Nope, you are correct. I think I made the update before converting to `&&` in the tasks.json and forgot to remove once the `&&` passed the error along because the change wasn't wrong (just unnecessary).